less/mixins.less

```diff
@@ -1 +1,19 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+@button-font-color: white;
+
+.button-colors{
+    color: @button-font-color;
+    background: @button-bg;
+}
+
```

Comment:
Good use of variables here. To do the parametric mixins for the buttons check out http://lesscss.org/features/#mixins-feature-mixins-parametric-feature